stingray/crossspectrum.py

```diff
@@ -1736,6 +1780,61 @@ def deadtime_correct(
 
         return new_spec
 
+    def _calculate_errors(self):
+        """Calculate the errors on cross powers and lags.
+
+        Uses the formulas from Ingram 2019, MNRAS 489, 392. The attribute ``channels_overlap``
```

Comment:
Could it be better to indicate the formulas (the ones Matteo Lucchini wrote in the PR) or is it too specific? Also, I remember that in one of the first implementation of Ingram's approach in Stingray, a link to the paper was provided, although here you already put the reference. I leave you the choice to put the link or leave the reference as it is.

Reply:
The formulas are a little complicated, but I did expand the description and add a link to the paper in the original function and referred to that function here, hope it is more clear now

Reply:
Yes indeed it is clearer now, better this way. Thank you Matteo